### ISSUE TITLE
Backend request doesn't contains cookie when cookie configured in browsertime

### ISSUE THREAD

**Issue by liranGlik** [sitespeedio/sitespeed.io#2185]
Hi, 
I trying to use browsertime's cookie parameter in order to send all the requests with this cookie.
The problem is, when printing the request at the server side, I'm not getting the cookie information.

the config.json looks like this:

```
{
  "browsertime": {
               ...
               "cookie": "blabla=1",
               ...
   }
   ...
}
```
- i'm testing it locally (macOS)
- in my Node.js package.json using version `"^7.3.4"`
- browser: chrome Version 69.0.3497.100 (Official Build) (64-bit)

any idea what went wrong?

**Comment by soulgalore** [comment]
Hi @liranGlik but you see it in the HAR? Let me try to reproduce it tonight.
Best
Peter

**Comment by soulgalore** [comment]
Hi @liranGlik this seems to work for me:
`docker run --shm-size=1g --rm -v "$(pwd)":/sitespeed.io sitespeedio/sitespeed.io:7.6.3 https://httpbin.org/cookies -n 1 --browsertime.cookie apa=hepp`

Best
Peter

**Comment by omrilotan** [comment]
Found the issue with Liran: The option `browsertime.cacheClearRaw` cleared the cookies set previously.
Is this considered expected behaviour or would you consider a "fix" for this @soulgalore ?

**Comment by soulgalore** [comment]
Thanks @omrilotan then it is a bug, the extension (that both clear the cache and add the cookie) does things in the wrong order then. I'll make a fix for this later today. Sorry!

Best
Peter

**Comment by soulgalore** [comment]
I checked the code and that's the problem. For Firefox we do a promise.all on all things that need to be done in the extension (but that's wrong) we should of course clear the cache first. For Chrome we just use the old callbacks instead of promised. I wanna clean this up with the fix, so it will take some time. I'll start tonight.

**Comment by omrilotan** [comment]
Sounds good. LMK if you want help

**Comment by omrilotan** [comment]
Moving this issue to the extension repo https://github.com/sitespeedio/browsertime-extension/issues/12

**Comment by soulgalore** [comment]
I've pushed a new version in master, thank you @omrilotan for the fix!

**Comment by soulgalore** [comment]
Released this early this morning in 7.7.0! I'll close the issue but please let me know if you still have a problem.

**Comment by soulgalore** [comment]
Released the fix in 7.7.0 early this morning, thank you again @omrilotan !
